src/legacy/core_plugins/elasticsearch/index.d.ts

```diff
@@ -523,6 +523,4 @@ export interface CallCluster {
 export interface ElasticsearchPlugin {
   status: { on: (status: string, cb: () => void) => void };
   getCluster(name: string): Cluster;
-  createCluster(name: string, config: ClusterConfig): Cluster;
-  waitUntilReady(): Promise<void>;
```

Comment:
let's remove this comment as well https://github.com/elastic/kibana/blob/bf04235dae35452061cc7ea3d86d96c19a58206c/src/core/server/ui_settings/integration_tests/index.test.ts#L35